### PR TITLE
check: make the user's duty to init consensus at the next height

### PR DESCRIPTION
**Pull request by AnnaShaleva**
It's OK to notify the user about the block collected and do not change the context's state immediately. The user will handle new block and call `InitializeConsensus` by itself like he already does when the new block comes from the other source (e.g. network/RPC). The consensus won't perform any further actions anyway before the next call to `InitializeContext`. The described way of responsibility distribution leads to clear and predictable behaviour and cleaner user's handlers code.

Moreover (it's not an argument, but yet another valid reason to apply this commit), the C# dBFT plugin does the same thing. It leaves the initialization to the `OnPersistCompleted` callback and does not initialize consensus at the new heigh immediately after Commits checks. See the https://github.com/neo-project/neo-modules/blob/52122bd286b0a9854ded648516bb223a8ecb4d2e/src/DBFTPlugin/Consensus/ConsensusService.Check.cs#L64 and
https://github.com/neo-project/neo-modules/blob/52122bd286b0a9854ded648516bb223a8ecb4d2e/src/DBFTPlugin/Consensus/ConsensusService.cs#L75.

It should be noted that it's still the dBFT's duty to initialize consensus at the next non-zero view (e.g. on collecting M ChangeViews), because it is the direct operation of the consensus algorithm.

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
OK, it needs some more care, the node is actually able to accept Commits after the block approval and notifies the blockchain one more time:
```
2023-03-09T10:27:44.661Z	INFO	approving block	{"height": 2695, "hash": "23fa715d248eb54170d688ef269d025464bf895a32cdcff1962664c616d5d612", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "a5450726bdbea99c949cfb13a04e620c1182155683650963b4e451f30ba0963c"}
2023-03-09T10:27:44.662Z	INFO	received Commit	{"validator": 3}
2023-03-09T10:27:44.662Z	INFO	approving block	{"height": 2695, "hash": "23fa715d248eb54170d688ef269d025464bf895a32cdcff1962664c616d5d612", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "a5450726bdbea99c949cfb13a04e620c1182155683650963b4e451f30ba0963c"}
```

**Comment by AnnaShaleva** [comment]
Finally, I've managed to find out the reason of this behaviour and fix the bug, @roman-khimov, you should take a look, it's a juicy one! The logs of privnet after this fix look pretty normal now, and I'm satisfied with the consensus behaviour with these fixes, it matches the expectations:
<details>
<summary>logs</summary>

```
2023-03-09T16:14:22.181Z	INFO	received PrepareResponse	{"validator": 1}
2023-03-09T16:14:22.181Z	INFO	sending Commit	{"height": 2820, "view": 0}
2023-03-09T16:14:22.182Z	INFO	received Commit	{"validator": 1}
2023-03-09T16:14:22.183Z	INFO	received Commit	{"validator": 0}
2023-03-09T16:14:22.183Z	INFO	approving block	{"height": 2820, "hash": "01496aea327281ef8963607aca52b21c2ee4c22de0a5557e9998d3f191636d9f", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "d4abd52bd2dfc3f196e7616d1d8c0bf69925a4cf77d2c31dd6996facf4254ff7"}
2023-03-09T16:14:22.184Z	INFO	initializing dbft	{"height": 2821, "view": 0, "index": 2, "role": "Backup"}
2023-03-09T16:14:22.219Z	INFO	new peer connected	{"addr": "172.200.0.3:20335", "peerCount": 3}
2023-03-09T16:14:22.220Z	INFO	started protocol	{"addr": "172.200.0.3:20335", "userAgent": "/NEO-GO:0.101.1-pre-39-g7b930b8d/", "startHeight": 2819, "id": 1276497129}
2023-03-09T16:14:22.567Z	INFO	new peer connected	{"addr": "172.200.0.3:41862", "peerCount": 4}
2023-03-09T16:14:22.568Z	WARN	peer disconnected	{"addr": "172.200.0.3:41862", "error": "already connected", "peerCount": 3}
2023-03-09T16:14:22.877Z	INFO	persisted to disk	{"blocks": 1, "keys": 20, "headerHeight": 2820, "blockHeight": 2820, "took": "3.706814ms"}
2023-03-09T16:14:37.187Z	INFO	received PrepareRequest	{"validator": 1, "tx": 0}
2023-03-09T16:14:37.187Z	INFO	sending PrepareResponse	{"height": 2821, "view": 0}
2023-03-09T16:14:37.189Z	INFO	received PrepareResponse	{"validator": 0}
2023-03-09T16:14:37.189Z	INFO	sending Commit	{"height": 2821, "view": 0}
2023-03-09T16:14:37.191Z	INFO	received Commit	{"validator": 1}
2023-03-09T16:14:37.191Z	INFO	received PrepareResponse	{"validator": 3}
2023-03-09T16:14:37.192Z	INFO	received Commit	{"validator": 3}
2023-03-09T16:14:37.192Z	INFO	approving block	{"height": 2821, "hash": "afb1b54de746754d08d59b446fc5bc1a5d825a928c5d55fdcd883d6bf01abd51", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "01496aea327281ef8963607aca52b21c2ee4c22de0a5557e9998d3f191636d9f"}
2023-03-09T16:14:37.193Z	INFO	initializing dbft	{"height": 2822, "view": 0, "index": 2, "role": "Primary"}
2023-03-09T16:14:37.886Z	INFO	persisted to disk	{"blocks": 1, "keys": 19, "headerHeight": 2821, "blockHeight": 2821, "took": "3.046234ms"}
2023-03-09T16:14:52.194Z	INFO	sending PrepareRequest	{"height": 2822, "view": 0}
2023-03-09T16:14:52.199Z	INFO	received PrepareResponse	{"validator": 0}
2023-03-09T16:14:52.199Z	INFO	received PrepareResponse	{"validator": 1}
2023-03-09T16:14:52.199Z	INFO	sending Commit	{"height": 2822, "view": 0}
2023-03-09T16:14:52.200Z	INFO	received PrepareResponse	{"validator": 3}
2023-03-09T16:14:52.201Z	INFO	received Commit	{"validator": 1}
2023-03-09T16:14:52.201Z	INFO	received Commit	{"validator": 0}
2023-03-09T16:14:52.202Z	INFO	approving block	{"height": 2822, "hash": "3ad4b0593abe312f173b43a33ec5c1fef49d107082b8b87c53442793f195786a", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "afb1b54de746754d08d59b446fc5bc1a5d825a928c5d55fdcd883d6bf01abd51"}
```

</details>

And in debug mode:

<details>
<summary>debug logs</summary>
```
2023-03-09T16:23:14.292Z	DEBUG	not enough to commit	{"count": 2}
2023-03-09T16:23:14.293Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.293Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.293Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:14.294Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:14.295Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:14.296Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDExtensible"}
2023-03-09T16:23:14.296Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:14.297Z	DEBUG	received message	{"from": 1, "type": "Commit"}
2023-03-09T16:23:14.297Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:14.297Z	DEBUG	received message	{"type": "Commit", "from": 1, "height": 2833, "view": 0, "my_height": 2833, "my_view": 0}
2023-03-09T16:23:14.297Z	INFO	received Commit	{"validator": 1}
2023-03-09T16:23:14.298Z	INFO	approving block	{"height": 2833, "hash": "ba87aeef9616ae1962000d1ea27c0cea86860869129482e47579a0eb9d7f5d79", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "0927edb259850f741f188896d2bb6cf45042d3e5f17621b9f840bfd799ed2d30"}
2023-03-09T16:23:14.298Z	DEBUG	received message	{"from": 3, "type": "Commit"}
2023-03-09T16:23:14.300Z	DEBUG	done processing headers	{"headerIndex": 2833, "blockHeight": 2832, "took": "1.803203ms"}
2023-03-09T16:23:14.301Z	DEBUG	new block in the chain	{"dbft index": 2833, "chain index": 2833}
2023-03-09T16:23:14.301Z	INFO	initializing dbft	{"height": 2834, "view": 0, "index": 2, "role": "Primary"}
2023-03-09T16:23:14.301Z	DEBUG	reset timer	{"h": 2834, "v": 0, "delay": "14.996109879s"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDGetData"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDExtensible"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:14.302Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:14.303Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:14.303Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:14.304Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.304Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:14.304Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:14.884Z	INFO	persisted to disk	{"blocks": 1, "keys": 19, "headerHeight": 2833, "blockHeight": 2833, "took": "6.087228ms"}
2023-03-09T16:23:29.298Z	DEBUG	timer fired	{"height": 2834, "view": 0}
2023-03-09T16:23:29.298Z	DEBUG	timeout	{"height": 2834, "view": 0}
2023-03-09T16:23:29.298Z	DEBUG	broadcasting message	{"type": "PrepareRequest", "height": 2834, "view": 0}
2023-03-09T16:23:29.299Z	INFO	sending PrepareRequest	{"height": 2834, "view": 0}
2023-03-09T16:23:29.299Z	DEBUG	reset timer	{"h": 2834, "v": 0, "delay": "15s"}
2023-03-09T16:23:29.299Z	DEBUG	check preparations	{"hasReq": true, "count": 1, "M": 3}
2023-03-09T16:23:29.300Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:29.300Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:29.300Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDGetData"}
2023-03-09T16:23:29.301Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.301Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.302Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.302Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.302Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.303Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.303Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDExtensible"}
2023-03-09T16:23:29.303Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDExtensible"}
2023-03-09T16:23:29.304Z	DEBUG	received message	{"from": 3, "type": "PrepareResponse"}
2023-03-09T16:23:29.304Z	DEBUG	received message	{"type": "PrepareResponse", "from": 3, "height": 2834, "view": 0, "my_height": 2834, "my_view": 0}
2023-03-09T16:23:29.304Z	INFO	received PrepareResponse	{"validator": 3}
2023-03-09T16:23:29.304Z	DEBUG	check preparations	{"hasReq": true, "count": 2, "M": 3}
2023-03-09T16:23:29.304Z	DEBUG	received message	{"from": 0, "type": "PrepareResponse"}
2023-03-09T16:23:29.304Z	DEBUG	received message	{"type": "PrepareResponse", "from": 0, "height": 2834, "view": 0, "my_height": 2834, "my_view": 0}
2023-03-09T16:23:29.304Z	INFO	received PrepareResponse	{"validator": 0}
2023-03-09T16:23:29.304Z	DEBUG	check preparations	{"hasReq": true, "count": 3, "M": 3}
2023-03-09T16:23:29.304Z	INFO	sending Commit	{"height": 2834, "view": 0}
2023-03-09T16:23:29.304Z	DEBUG	broadcasting message	{"type": "Commit", "height": 2834, "view": 0}
2023-03-09T16:23:29.304Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	reset timer	{"h": 2834, "v": 0, "delay": "15s"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:29.305Z	DEBUG	not enough to commit	{"count": 1}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDExtensible"}
2023-03-09T16:23:29.305Z	DEBUG	received message	{"from": 1, "type": "PrepareResponse"}
2023-03-09T16:23:29.305Z	DEBUG	received message	{"type": "PrepareResponse", "from": 1, "height": 2834, "view": 0, "my_height": 2834, "my_view": 0}
2023-03-09T16:23:29.305Z	INFO	received PrepareResponse	{"validator": 1}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDGetData"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDExtensible"}
2023-03-09T16:23:29.305Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDExtensible"}
2023-03-09T16:23:29.306Z	DEBUG	received message	{"from": 3, "type": "Commit"}
2023-03-09T16:23:29.306Z	DEBUG	received message	{"type": "Commit", "from": 3, "height": 2834, "view": 0, "my_height": 2834, "my_view": 0}
2023-03-09T16:23:29.306Z	INFO	received Commit	{"validator": 3}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDGetData"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDExtensible"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	not enough to commit	{"count": 2}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:29.306Z	DEBUG	received message	{"from": 0, "type": "Commit"}
2023-03-09T16:23:29.306Z	DEBUG	received message	{"type": "Commit", "from": 0, "height": 2834, "view": 0, "my_height": 2834, "my_view": 0}
2023-03-09T16:23:29.306Z	INFO	received Commit	{"validator": 0}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:29.306Z	INFO	approving block	{"height": 2834, "hash": "736c653c504be925ce83c4a98e8e3fff817f150f7e4df279d60dd91ad2914042", "tx_count": 0, "merkle": "0000000000000000000000000000000000000000000000000000000000000000", "prev": "ba87aeef9616ae1962000d1ea27c0cea86860869129482e47579a0eb9d7f5d79"}
2023-03-09T16:23:29.306Z	DEBUG	received message	{"from": 1, "type": "Commit"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDExtensible"}
2023-03-09T16:23:29.306Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.307Z	DEBUG	done processing headers	{"headerIndex": 2834, "blockHeight": 2833, "took": "439.663µs"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDInv"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDInv"}
2023-03-09T16:23:29.307Z	DEBUG	new block in the chain	{"dbft index": 2834, "chain index": 2834}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.4:48652", "type": "CMDBlock"}
2023-03-09T16:23:29.307Z	INFO	initializing dbft	{"height": 2835, "view": 0, "index": 2, "role": "Backup"}
2023-03-09T16:23:29.307Z	DEBUG	reset timer	{"h": 2835, "v": 0, "delay": "29.999141716s"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.1:20334", "type": "CMDInv"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDGetData"}
2023-03-09T16:23:29.307Z	DEBUG	got msg	{"addr": "172.200.0.3:20335", "type": "CMDBlock"}
2023-03-09T16:23:29.897Z	INFO	persisted to disk	{"blocks": 1, "keys": 19, "headerHeight": 2834, "blockHeight": 2834, "took": "4.075442ms"}
```
</details>

File: dbft.go
Comment (roman-khimov) [inline dbft.go:496]: The node committed at view 0 can't commit at view 1. Well, I know it technically is correct for dbft 2.0.

File: dbft.go
Comment (roman-khimov) [inline dbft.go]: Suppose you're the only node with this block. Now you get a recovery request from someone else.

**Review by roman-khimov — COMMENTED** [review]
I'll need to check this wrt neo-bench, although it should be OK. But it seems to be legit otherwise.

File: dbft.go
Comment (AnnaShaleva) [reply dbft.go:496]: Yes, we'll need to review the whole dbft code *very carefully* on the way to 2.1. There's a set of snippets that are valid for 2.0 but invalid for 2.1. I'd suggest to keep this change for now. Or not?

File: dbft.go
Comment (AnnaShaleva) [reply dbft.go]: Well, we have server for that. The server will relay our block, so thet if it can be delivered to the other node, then it doesn't need recovery request. And if it can't, then the recovery request delivering will likely fail as far.

But OK, it's a valid question that I kept in mind as far, so let's ignore everything except the recovery requests.

Comment (AnnaShaleva) [reply dbft.go]: Fixed.

File: dbft.go
Comment (roman-khimov) [reply dbft.go:496]: We keep it, it makes sense for 2.0.

File: dbft.go
Comment (roman-khimov) [inline dbft.go]: Huh.

**Review by roman-khimov — APPROVED** [review]

File: dbft.go
Comment (AnnaShaleva) [reply dbft.go]: Lovely linter, fixed.

**Pull request merged**
